src/lava/lib/optimization/solvers/generic/nebm/models.py

```diff
@@ -0,0 +1,130 @@
+import numpy as np
+
+from lava.lib.optimization.solvers.generic.nebm.process import NEBM
+from lava.magma.core.decorator import implements, requires, tag
+from lava.magma.core.model.py.model import PyLoihiProcessModel
+from lava.magma.core.model.py.ports import PyInPort, PyOutPort
+from lava.magma.core.model.py.type import LavaPyType
+from lava.magma.core.resources import CPU
+from lava.magma.core.sync.protocols.loihi_protocol import LoihiProtocol
+
+
+@implements(proc=NEBM, protocol=LoihiProtocol)
+@requires(CPU)
+@tag('fixed_pt')
+class NEBMPyModel(PyLoihiProcessModel):
+    """Fixed point implementation of Stochastic Constraint Integrate and
+        Fire (SCIF) neuron for solving QUBO problems.
+    """
+    a_in = LavaPyType(PyInPort.VEC_DENSE, int, precision=8)
+    s_sig_out = LavaPyType(PyOutPort.VEC_DENSE, int, precision=8)
+    s_wta_out = LavaPyType(PyOutPort.VEC_DENSE, int, precision=8)
+
+    # The local cost
+    state: np.ndarray = LavaPyType(np.ndarray, int, precision=24)
+    spk_hist: np.ndarray = LavaPyType(np.ndarray, int, precision=8)
+
+    temperature: np.ndarray = LavaPyType(np.ndarray, int, precision=8)
+    refract: np.ndarray = LavaPyType(np.ndarray, int, precision=8)
+
+    def __init__(self, proc_params):
+        super(NEBMPyModel, self).__init__(proc_params)
```

Comment:
In Python 3, you can just call
`super().__init__(proc_params)`
without providing arguments to super.